v2/internal/pkg/cli/executor.go

```diff
@@ -359,13 +354,13 @@ func (o *ExecutorSchema) Complete(args []string) error {
 		o.Log.Debug("destination %s ", rootDir)
 		// destination is the local cache, which is HTTP
 		// nolint: errcheck
-		o.destFlagSet.Set("dest-tls-verify", "false")
+		o.Opts.DestImage.TlsVerify = false
```

Comment:
@aguidirh - much better, I was also looking at this for the rebuild catalogs stuff, I found it ugly to have to do a key/value set for a simple true false flag